### PR TITLE
test

### PR DESCRIPTION
**Pull request by vanestw**

### CODE REVIEW

**Comment by clamsfeel2** [comment]
What the fuck is this?

**Comment by vanestw** [comment]
> What the fuck is this?

fork you

**Comment by clamsfeel2** [comment]
This is a terrible request. CLOSED.

File: hello.cpp [hunk 1]
```diff
@@ -1,21 +1,2 @@
-#include <iostream>
-#include <string>
-#include <vector>
 
-using namespace std;
-
-int main() {
-    cout << "fuck you" << endl;
```
Comment (clamsfeel2) [inline hello.cpp:8]: Please use more school appropriate language.

**Comment by vanestw** [comment]
why are you closing things on my bitch

**Pull request closed without merging**